src/stories/theme.js

```diff
@@ -17,5 +17,23 @@ const theme = {
   badgeBg: "#FF3D00",
   badgeText: "#fff",
   white: "#f6f9fc",
+  small:{
+    width:'50px',
+    height:'50px'
+},
+medium:{
+    width:'60px',
+    height: '60px'
+}
+,
+
+large:{
+    width:'60px',
+    height:'80px'
```

Comment:
use same as width @PRABgit01

Reply:
sir this is extra file i have done this changes in style only